### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by VICTOR-CHUKWU**
Use React components.
Use React props.
Use React Router.
Connect React and Redux.
Handle events in a React app.
Use styles in a React app.
Apply React best practices and language style guides in code.
Use store, actions, and reducers in React.
display cryptos
on click takes you to a detail page

### CODE REVIEW

File: src/Service.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+//     },
+//     json: true,
+//     gzip: true,
+//   });
+//   const data = await resp.json();
+//   console.log(data, 'data');
+//   return data;
+// };
+
+// export default fetchCrypto;
```
Comment (Emmyn5600) [inline src/Service.js]: - [ ] Let us remove the commented code in the code that has been pushed.

File: src/shared/Box.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/no-children-prop */
+/* eslint-disable react/prop-types */
+import {
+  useState, useEffect, memo, createElement,
+} from 'react';
+
```
Comment (Emmyn5600) [inline src/shared/Box.js:6]: - [ ] Let us fix the eslint errors without disabling them.

File: src/component/home/CryptoData.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+/* eslint-disable camelcase */
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
```
Comment (Emmyn5600) [inline src/component/home/CryptoData.js]: - [ ] Again, let us fix the eslint errors without disabling them.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi, @VICTOR-CHUKWU 

Highlights:

- Your project is running successfully,
- Linters are passing successfully.
- You have completed almost all the requirements.
- You have a descriptive readme file.
- Your project looks awesome, I liked the design.

Great work on making the project requirements:clap:
You've done well implementing the project requirements, but there are still some issues which aren't been addressed yet 
 
## STATUS: CHANGES REQUESTED :recycle:

### Changes requested.

- [ ] Check the comment under the review.
- [ ] ##### **Testing**

1. App has unit tests for several components of the app and some pure functions in Redux code.
1. App has integration tests using React Testing Library
   * Student can perform a mocking data fetching and Redux Store connection.
   
   Please implement the tests

 #### Optional suggestions
 
 - Please make more commits while you are working on the project as well as milestones.
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by VICTOR-CHUKWU** [comment]
The three linters disabled, two in Box.js and one in CryptoData.js is actually necessary to achieve what I intend to do. thanks

**Comment by VICTOR-CHUKWU** [comment]
the last reviewer did not leave any comment, but I saw changes required in my dashboard, kindly assist

**Comment by Emmyn5600** [comment]
The last reviewer, what do you mean? Anyway you are supposed to work on testing

**Review by AshakaE — APPROVED** [review]
### Hi @VICTOR-CHUKWU   🙋‍♀️, 

# STATUS APPROVED ✔
  
Thanks for making the requested changes 👏 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VICTOR-CHUKWU** [comment]
> 

It was not you, I got another review after yours, but the reviewer did not comment

**Pull request merged**